jmix-flowui/flowui-kit/src/main/resources/META-INF/frontend/theme/lumo/checkbox/jmix-switch-styles.js

```diff
@@ -74,7 +75,7 @@ registerStyles('jmix-switch',
             margin: var(--lumo-space-xs);
 
             position: relative;
-            border-radius: var(--jmix-switch-border-radius, var(--lumo-border-radius-l));
+            border-radius: var(--jmix-switch-border-radius, calc(var(--jmix-switch-size) / 2));
```

Comment:
1. you need to refer to `--_switch-size` instead
2. Set `--lumo-tint-90pct` as the default value for `--jmix-switch-indicator-color`. It suits both light and dark themes